assets/scss/_styles_project.scss

```
@@ -987,52 +987,30 @@ td > ul, td > ol {
   }
 }
 
+// relatedcard styling
 
-// relatedlist styling
-
-ul.relatedlist {
-  padding: 0;
+.relatedcard {
   display: flex;
+  flex-direction: column;
 }
 
-ul.relatedlist > li {
-  list-style-type: none;
-  padding-right: 2rem;
-}
-
-ul.relatedlist > li > a {
+.relatedcard a {
+  width: 7rem;
```

Data Management is currently just a bit too wide
```suggestion
  width: 8rem;
```